pylutron_caseta/smartbridge.py

```diff
@@ -143,6 +144,10 @@ def get_devices(self) -> Dict[str, dict]:
         """Will return all known devices connected to the Smart Bridge."""
         return self.devices
 
+    def get_lip_devices(self) -> Dict[int, dict]:
```

Comment:
If `lip_devices` is unprefixed, do we also need `get_lip_devices`?

Reply:
Nope, I'll take it out.  It was only there to match `get_devices`